src/main/resources/templates/forms_hdc_address.html

```diff
@@ -4,10 +4,12 @@
     <title>HDC Eligible</title>
     <style type="text/css">
         @page {
+            size: letter;
```

Comment:
Force all pages to be us letter as new default is A4.  Obviously this is wrong, but for now be consistent with existing pages and remove with new JIRA ticket at later point.